### PR TITLE
XF-635 | separate endpoints for wiki variables used by mobile-wiki, discussions and announcements

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XF-635

@Wikia/iwing

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -180,6 +179,48 @@ private function prepareWikiVariables() {
 		return $wikiVariables;
 	}
 
+	public function getMobileWikiVariables() {
+		( new CrossOriginResourceSharingHeaderHelper() )->allowWhitelistedOrigins()
+			->setAllowMethod( [ 'GET' ] )
+			->setHeaders( $this->response );
```
Comment (mszabo-wikia) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:80]: is it needed here? If it's internal traffic only I think we can remove it

Comment (rybmat) [reply extensions/wikia/MercuryApi/MercuryApiController.class.php:80]: @mszabo-wikia requests to these endpoints come also from front-end apps

Comment (rogatty) [reply extensions/wikia/MercuryApi/MercuryApiController.class.php:80]: @rybmat which ones? I thought all Ember clients use a server-side request.

Comment (rybmat) [reply extensions/wikia/MercuryApi/MercuryApiController.class.php:80]: @rogatty I'll double check that, thanks!

**Pull request merged**